### ISSUE TITLE
Add support for custom exe

### ISSUE THREAD

**Issue by rohanmohapatra** [spectresystems/jarvis#98]
I have an .exe present in my downloads folder, not able to index it using Jarvis

**Comment by patriksvensson** [comment]
@rohanmohapatra Have you added the downloads folder to the list of indexed locations?

**Comment by rohanmohapatra** [comment]
Yes, but the folder gets indexed not the exe present inside the folder
